radis/test/misc/test_utils.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+import pytest
+
+
+def test_files_from_regex(*args, **kwargs):
+    from radis.misc.utils import get_files_from_regex
+    from radis.test.utils import getTestFile
+
+    path = "/home/aditya/radis/radis/test/files/*.par"
+
+    path = get_files_from_regex(path)
+
+    for files in path:
+        print(getTestFile(files))
+
```

Comment:
what are you testing here? You should add assertions.  For instance that you correctly found all the *.par files of the test files.

Reply:
Added asserts